llvm/test/CodeGen/AArch64/addsub_ext.ll

```diff
@@ -24,22 +24,12 @@ define i32 @add_z_shli8i32(i8 %v, i32 %lhs) minsize {
 }
 
 define i64 @add_z_i8i64(i8 %v, i64 %lhs) minsize {
-; CHECK-LABEL: add_z_i8i64:
-; CHECK:       // %bb.0:
```

Comment:
This lost all the checks (this is a really bad update_llc_test_checks bug when this happens)